eaf.el

```diff
@@ -1749,7 +1749,7 @@ For a full `install-eaf.py' experience, refer to `--help' and run in a terminal.
       (byte-recompile-directory eaf-build-dir 0))
   (message "Cleaning load path")
   (setq load-path
-	(cl-remove-if (lambda (s) (string-prefix-p eaf-build-dir s)) load-path))
+	(cl-remove-if (lambda (s) (string-prefix-p (directory-file-name eaf-build-dir) s)) load-path))
```

Comment:
Use let expression move directory-file-name out of loop, will improve performance.